flattentool/input.py

```diff
@@ -397,7 +412,17 @@ def get_sheet_headings(self, sheet_name):
 
     def read_sheets(self):
         sheet_file_names = os.listdir(self.input_name)
-        self.sub_sheet_names = sorted([fname[:-4] for fname in sheet_file_names if fname.endswith('.csv')])
+        sheet_names = sorted([fname[:-4] for fname in sheet_file_names if fname.endswith('.csv')])
+        if self.include_sheets:
+            for sheet in list(sheet_names):
+                if sheet not in self.include_sheets:
```

Comment:
Should this be case insensitive? The metatab spec isn't consistent with case for "Meta"